<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial Key Generator | https://laroussigsm.net/</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ico.qrc">
    <normaloff>:/newPrefix/Icons/password.png</normaloff>:/newPrefix/Icons/password.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="Container">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>671</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame#Container{
background-color: rgb(28, 40, 51);
}

QLineEdit {
border:1px solid qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(45, 45, 45, 255), stop:1 rgba(97, 130, 171, 255));
border-radius:5px;	
color: rgb(247, 247, 247);
padding-right:5px;
padding-left:5px;
background-color: rgb(27, 38, 49);
}

QGroupBox{
color: rgb(255, 255, 255);
border:1px solid qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(45, 45, 45, 255), stop:1 rgba(97, 130, 171, 255));
border-radius:5px;
}

QTextEdit {

border:1px solid qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(45, 45, 45, 255), stop:1 rgba(97, 130, 171, 255));
border-radius:5px;	
color: rgb(247, 247, 247);
padding-right:5px;
padding-left:5px;
background-color: rgb(27, 38, 49);
}

QLabel{
color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>571</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="title">
      <string>Hardware ID </string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <widget class="QLineEdit" name="HID_lineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>511</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(229, 255, 241);
color: rgb(12, 12, 12);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="Copy_id_Btn">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>20</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
  background-color: rgb(28, 40, 51);
  color: rgb(255, 255, 255);
  border-radius: 3px;
}
QPushButton::pressed{
	padding-left:2px;
	padding-top: 1px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="ico.qrc">
        <normaloff>:/newPrefix/Icons/files.png</normaloff>:/newPrefix/Icons/files.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>571</width>
       <height>161</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Registration Information </string>
     </property>
     <widget class="QLineEdit" name="Name_lineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>511</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>15</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Key</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="Key_textEdit">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>100</y>
        <width>511</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Name_Paste_Btn">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>40</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
  background-color: rgb(28, 40, 51);
  color: rgb(255, 255, 255);
  border-radius: 3px;
}
QPushButton::pressed{
	padding-left:2px;
	padding-top: 1px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="ico.qrc">
        <normaloff>:/newPrefix/Icons/paste.png</normaloff>:/newPrefix/Icons/paste.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="Copy_Key_Btn">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>100</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
  background-color: rgb(28, 40, 51);
  color: rgb(255, 255, 255);
  border-radius: 3px;
}
QPushButton::pressed{
	padding-left:2px;
	padding-top: 1px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="ico.qrc">
        <normaloff>:/newPrefix/Icons/files.png</normaloff>:/newPrefix/Icons/files.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="Generate_Btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
  background-color: rgb(9, 132, 227);
  color: rgb(255, 255, 255);
  border-radius: 5px;
}
QPushButton::hover{
	background-color:rgb(85, 161, 227);
}
QPushButton::pressed{
	padding-left:2px;
	padding-top: 1px;
	background-color: rgb(223, 160, 5);
}</string>
     </property>
     <property name="text">
      <string> Generate Activation Code</string>
     </property>
     <property name="icon">
      <iconset resource="ico.qrc">
       <normaloff>:/newPrefix/Icons/key.png</normaloff>:/newPrefix/Icons/key.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="Close_Btn">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>250</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
  background-color: rgb(214, 48, 49);
  color: rgb(255, 255, 255);
  border-radius: 5px;
}
QPushButton::hover{
	background-color:rgb(209, 109, 114);
}
QPushButton::pressed{
	padding-left:2px;
	padding-top: 1px;
	background-color: rgb(223, 160, 5);
}</string>
     </property>
     <property name="text">
      <string> Close</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="ico.qrc"/>
 </resources>
 <connections/>
</ui>
